Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -107,6 +107,14 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "Specify the name of the Virtual Application that has been configured in the Azure portal. The option is not required for deployments to the website root."
+         },
+        {
+            "name": "AdditionalArguments",
+            "type": "string",
+            "label": "Additional Arguments",
+            "required": false,
+            "defaultValue": "",
+            "helpMarkDown": "Additional MSDeploy.exe arguments that will be applied while uploading asp.net website."
```

Comment:
please get it reviewed by ambrish once meanwhile you can change "Additional MSDeploy.exe arguments that will be applied while deploying web app" dont restrict type of website in a help markdown yet otherwise we have to tell which version also we support which we will capture later in read.me.